test/extended/olm/olmv1.go

```diff
@@ -99,6 +101,43 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLM
 	})
 })
 
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 Catalog's /v1/api/all endpoint", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("default")
+
+	g.It("should serve FBC", func(ctx g.SpecContext) {
+		checkFeatureCapability(ctx, oc)
+
+		// Start port-forward in background
+		_, _, _, err := oc.AsAdmin().Run("port-forward").Args("-n", "openshift-catalogd", "svc/catalogd-service", "8080:443").Background()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// Wait for port-forward to be established
+		time.Sleep(2 * time.Second)
```

Comment:
The port-forward isn't ready by the time the `oc.Run("port-forward")` command returns?

Reply:
I was just being extra cautious